docs/Language Feature Status.md

```diff
@@ -16,7 +16,7 @@ efforts behind them.
 | [Infer tuple names](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-7.1/infer-tuple-names.md) | master | Merged | [jcouv](https://github.com/jcouv) | [gafter](https://github.com/gafter) | [jcouv](https://github.com/jcouv) |
 | [Pattern-matching with generics](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-7.1/generics-pattern-match.md) | master  | Merged | [gafter](https://github.com/gafter) | [agocke](https://github.com/agocke) | [gafter](https://github.com/gafter) |
 
-# C# 7.2
+# C# 7.2 (Roslyn master)
```

Comment:
I'd rather not add that. Branches are pretty fluid and master won't be C# 7.2 very shortly (a couple of weeks).